core/src/modules/caches/appsCache.ts

```diff
@@ -0,0 +1,52 @@
+import { ModelCache } from "../modelCache";
+import { App } from "../../models/App";
+import { Option } from "../../models/Option";
+import { Includeable, WhereAttributeHash } from "sequelize";
+
+function getInclude() {
+  const include: Includeable[] = [{ model: Option, required: false }];
+  return include;
+}
```

Comment:
Seems pretty simple to be its own function.

Reply:
I moved this into the constructor!